sdk/scheduler/src/main/java/com/mesosphere/sdk/state/GoalStateOverride.java

```diff
@@ -20,6 +21,15 @@
     /** The definition of the "STOPPED" override state, where commands are replaced with sleep()s.*/
     STOPPED("STOPPED", "STOPPING");
 
+    // Sleep for 2 weeks when pausing
+    public static final String PAUSE_COMMAND =
+            String.format(
+                    "echo This task is PAUSED, sleeping ... && " +
+                            "./bootstrap --resolve=false && while true; do sleep %d; done",
+                    Constants.LONG_DECLINE_SECONDS);
+
+    public static final String PAUSE_READINESS_COMMAND = "exit 1";
```

Comment:
Maybe leave a comment explaining the reasoning for doing this in the readiness command.